src/azure-cli-core/azure/cli/core/commands/client_factory.py

```diff
@@ -51,13 +51,18 @@ def configure_common_settings(client):
 
 
 def _get_mgmt_service_client(client_type, subscription_bound=True, subscription_id=None,
-                             api_version=None):
+                             api_version=None, base_url_bound=True, **kwargs):
```

Comment:
I could make it part of kwargs, but because it is used below, and there is a subscription_bound parameter, I figured it would be nice to make it more explicit for readability for people investigating this function.